diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/generation/Generation.kt

```diff
@@ -47,6 +65,37 @@ private fun generateWarningNames() {
     kotlinFile.writeTo(File("diktat-rules/src/main/kotlin"))  // fixme: need to add it to pom
 }
 
+@Suppress("MagicNumber")
+private fun generateAvailableRules() {
+    val ruleMap = File("info/rules-mapping.md").readLines().drop(2)
+            .map { it.drop(1).dropLast(1).split("|") }
+            .map { RuleDescription(it[0].replace("\\s+".toRegex(), ""), it[1], it[2], null) }
+            .map { it.ruleName to it }.toMap()
+    File("info/available-rules.md").readLines()
+            .drop(2)
+            .map { it.drop(1).dropLast(1).split("|") }
+            .map { it[2].replace("\\s+".toRegex(), "") to it[5] }
+            .forEach { ruleMap[it.first]?.config = it.second }
```

Comment:
I think you should either log a warning if `it.first` is not present in a map, or throw an error, if it means docs are inconsistent with each other.